CHANGELOG.md

```diff
@@ -1,7 +1,48 @@
 # Changelog
 
-Due to the rapidly changing nature of ETH2 testnets and rapid rate of improvements to Teku, 
-we recommend most users use the latest `master` branch of Teku.
+## Timeline to Mainnet
+
+With the announcement of the Eth2 deposit contract and setting the minimum genesis time to 
+Tuesday, December 1, 2020 12:00:00 PM UTC the journey to launching the beacon chain is in its final stage.
+Teku is ready to be a part of the Mainnet launch and we are working with other client teams to coordinate 
+all the details required to make the launch a success. Here's a timeline of key events, including the
+Teku releases that will lead up to Mainnet launch.
+
+Note that the exact genesis time depends on when enough deposits are received. If it takes longer
+to reach the minimum validators required, then the date the Mainnet genesis is set will be delayed 
+until enough validators are registered.  The Mainnet chain will then start 7 days after the genesis 
+is set. There will be a Teku release in the 7 days prior to the Mainnet chain starting and users 
+should ensure they upgrade to it.
```

Comment:
```suggestion
**Note**: The exact genesis time depends on when enough deposits are received. If it takes longer to reach the minimum validators required, then the Mainnet genesis date will be delayed until enough validators are registered. If delayed, then the Mainnet chain will start 7 days after genesis is set. There will be a Teku release within the 7 days before the Mainnet chain starts, and users should ensure they upgrade to it.
```

Reply:
The main net chain will always start 7 days after the genesis state is set, regardless of whether it is delayed or not.  Changing "the date the Mainnet genesis is set" to "the Mainnet genesis date" is unfortunately ambiguous - the genesis date is actually the date the chain starts, the genesis state is set (or becomes known) 7 days prior to that.

I think I've picked up as much of the the improvements to wording as I can...